contents/faq-page/faq.yml

```diff
@@ -312,6 +312,10 @@ questions:
         content: "Al termine del processo di rilascio del sito istituzionale per le scuole, l’ente dovrà attestare mediante dichiarazione da caricare sulla piattaforma di adesione https://padigitale2026.gov.it/ il raggiungimento di alcune caratteristiche, che rappresentano ai fini dell'Avviso riferito alla Misura 1.4.1, la dichiarazione di conformità del sito per le scuole realizzato rispetto al modello fornito.<br>In particolare, le attività si intendono concluse con esito positivo al momento in cui:<br>- Il sito è disponibile online<br>- È stata compilata la checklist di conformità indicata nel paragrafo Verifica della conformità dell'Allegato 2 all'avviso"
       - title: A quanto ammonta l'importo del finanziamento previsto dall'Avviso riferito alla Misura 1.4.1 “Esperienza del cittadino nei servizi pubblici” SCUOLE (Aprile 2022)”?
         content: L’importo del finanziamento, riconosciuto per il rifacimento del sito web istituzionale è pari a 7.301 euro Iva inclusa. L’importo del finanziamento sarà riconosciuto al Soggetto Attuatore solo a seguito del conseguimento del risultato atteso.
+      - title: A quanto ammonta l'importo del finanziamento previsto dall'Avviso riferito alla Misura 1.4.1 “Esperienza del cittadino nei servizi pubblici” SCUOLE (Aprile 2022)”?
+        content: L’importo del finanziamento, riconosciuto per il rifacimento del sito web istituzionale è pari a 7.301 euro Iva inclusa. L’importo del finanziamento sarà riconosciuto al Soggetto Attuatore solo a seguito del conseguimento del risultato atteso.
```

Comment:
seems they are duplicated

Reply:
Marica Wrote this:
_1.4.3 pagoPA + 1.4.3 appIO

D: Ho integrato dei servizi tramite una piattaforma regionale a titolo gratuito. Posso richiedere il finanziamento per l'integrazione nella stessa piattaforma dei servizi aggiuntivi?

R: Sì, è possibile accedere al finanziamento previsto dall'Avviso per l'implementazione dei servizi aggiuntivi purché siano finanziati con fondi propri dopo il 1 aprile 2021._

So I thought that FAQ was must be inserted for both

Reply:
I removed the duplicated